content/en/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping.md

```diff
@@ -197,19 +370,74 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ```
 
+
+The `csrapproving` controller that ships as part of
+[kube-controller-manager](/docs/admin/kube-controller-manager/) and is enabled
+by default. The controller uses the [`SubjectAccessReview`
+API](/docs/reference/access-authn-authz/authorization/#checking-api-access) to
+determine if a given user is authorized to request a CSR, then approves based on
+the authorization outcome. To prevent conflicts with other approvers, the
+builtin approver doesn't explicitly deny CSRs. It only ignores unauthorized
+requests. The controller also prunes expired certificates as part of garbage
+collection.
+
+
 ## kubelet configuration
+Finally, with the master nodes properly set up and all of the necessary authentication and authorization in place, we can configure the kubelet.
+
+The kubelet requires the following configuration to bootstrap:
 
-To request a client certificate from kube-apiserver, the kubelet first needs a
-path to a kubeconfig file that contains the credentials for the identity that it
-will use to bootstrap its individual node identity.
+* A path to store the key and certificate it generates (optional, can use default)
+* A path to a `kubeconfig` file that does not yet exist; it will place the bootstrapped config file here
+* A path to a bootstrap `kubeconfig` file to provide the bootstrap token and URL for the server
```

Comment:
`to provide the bootstrap credentials` ... don't imply the bootstrap credential must be a token

Reply:
Fair.